integration-test/helpers/pages.js

```diff
@@ -486,19 +486,25 @@ export function loginPage (page, opts = {}) {
             }
         },
         /**
-         * This is here to capture EXISTING functionality of `macOS` in production and prevent
-         * accidental changes to how `showAutofillParent` messages are sent
          * @returns {Promise<void>}
          */
-        async assertClickAndFocusMessages () {
+        async assertClickMessage () {
```

Comment:
This stuff now only fires one message, either the click or focus, so I've updated the test cases.